### PR TITLE
fixing dependency names for .NET PHP and python

### PR DESCRIPTION
**Pull request by slorello89**
## Description

Some of the code snippets for .NET / PHP / and Python were displaying the wrong dependency - updating

### CODE REVIEW

**Comment by dragonmantank** [comment]
PHP ones look good after pushing a fix that missed a few. Do we need to check the other three languages as well?

**Comment by slorello89** [comment]
Spot check only showed those three with issues - worth a look.

**Comment by slorello89** [comment]
@dragonmantank - I did a search of the entire `_examples` directory and replaced every reference to a nexmo package inside of a yml file

**Comment by mheap** [comment]
Is https://github.com/Nexmo/nexmo-developer/blob/master/config/code_languages.yml#L55 still correct? Or is the new package name `Vonage`?

File: _examples/messaging/sms/send-an-sms-with-unicode/php.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: PHP
 language: php
 dependencies:
-    - nexmo
+    - vonage
```
Comment (dragonmantank) [inline _examples/messaging/sms/send-an-sms-with-unicode/php.yml:5]: This should be `vonage/client`

**Review by dragonmantank — CHANGES_REQUESTED** [review]
Looks like PHP just had one incorrect one, otherwise the PHP snippets look good

File: _examples/messaging/sms/send-an-sms-with-unicode/php.yml (hunk 1 above)
Comment (slorello89) [reply _examples/messaging/sms/send-an-sms-with-unicode/php.yml]: fixed

**Review by dragonmantank — APPROVED** [review]
Good for my snippets

**Comment by tbedford** [comment]
@dragonmantank Can we merge?

**Comment by tbedford** [comment]
@lornajane @superdiana - are we good to go on this one?

**Comment by lornajane** [comment]
I'm not sure what I'm doing here :) I can't see that these changes would cause a problem, but I don't think they caught everything.

**Review by lornajane — APPROVED** [review]
Definitely a bunch of improvements here! Thanks people

**Pull request merged**